src/test/resources/processor/chunker/PipelineForFixedCharLengthChunker.json

```diff
@@ -0,0 +1,18 @@
+{
+  "description": "An example fixed character length chunker pipeline with overlap_rate",
```

Comment:
@yuye-aws ,
this json file is only for `fixed_char_length` algorithm. For integration-testing cascaded chunkers, I think it's better to rename the current file(`PipelineForCascadedChunker.json`) to `PipelineForCascadedChunkerWithDelimiterAndFixedTokenLength.json`, and create another one like `PipelineForCascadedChunkerWithDelimiterAndFixedCharLength.json`. Let me know your thoughts!